content/post/community-india-is-back.md

```diff
@@ -1,10 +1,50 @@
 +++
-author = ""
+author = "Dhruv Aggarwal"
 date = 2020-08-21T18:50:40Z
-description = ""
+description = "AWS Community Days are community-organized cloud education events, featuring technical discussions and demos led by expert AWS users and industry leaders from around the world."
 feature_image = ""
-tags = []
+tags = ["AWS"]
 title = "Welcome to AWS Community Day India, Virtual Edition"
 
 +++
-Welcome back! We're glad to have Community Day going again.
\ No newline at end of file
+We are glad to announce **AWS Community Day 2020, India Virtual Edition** from **16th - 17th October 2020** being organized by various AWS User Groups in India.
+
+![](/images/ug-logo-for-profile-pic-dark.png)
+
+**AWS User Group India** is united by community-led learning and diversified by the wide variety of tools and technologies closely knit with Amazon Web Services, cloud computing technology, AWS design implementation and servicing, high scalability / performance computing, production use cases of AWS, block chain, Artificial Intelligence and much more.
+
+Community-based learning is the best way to enhance the concepts being taught by connecting with enthusiasts in person, thereby sharing first-hand experiences and familiar, accessible examples.
+
+All skill levels are welcome to help accomplish our motto of learning, sharing and networking. Let's join hands to empower this tech community to make it bigger and better!
+
+## **WHAT TO EXPECT FROM AWS COMMUNITY DAY?**
+
+* Renew, revive & revitalize to exemplify your excitement about the work you do.
```

Comment:
can we rewrite this section? it's the exact copy+paste from the website

Reply:
Changed!